docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -7,17 +7,45 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-This document describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC and Mutual TLS authentication as well as the proactive authentication.
+The following section describes the Quarkus built-in authentication mechanisms for HTTP-based BASIC, FORM, and both Mutual TLS and proactive authentication.
 
 [[basic-auth]]
 == Basic Authentication
+ 
+HTTP Basic authentication (BA) uses fields in the HTTP header, and is the easiest to set up and least resource demanding technique for enforcing access controls to Web resources without requiring HTTP cookies, session identifiers, or login pages.
+
+In the context of an HTTP request, basic access authentication is a method for an HTTP user agent (such as a Web browser) to provide a user name and password when making a request. In basic HTTP authentication, a request contains a header field in the form of _Authorization: Basic_ _<credentials>_, where credentials are the Base64 encoding of a user ID and password joined by a colon. 
+
+====
+.Example
+ 
+If the user name is `Alice` and the password is `secret`, the HTTP Authorization header will look as `Authorization: Basic  QWxjZTpzZWNyZXQ=`,where `QWxjZTpzZWNyZXQ=` is a Base64 encoded representation of the `Alice:secret; string.
+====
+ 
+The BA mechanism does not provide confidentiality protection for the transmitted credentials; they are merely encoded with Base64 when in transit, and not encrypted or hashed in any way. Therefore, basic authentication is typically used with HTTPS to provide confidentiality.
+ 
+[[enabling-basic-auth]]
+=== Enabling Basic Authentication
+ 
+.Prerequisites
```

Comment:
Same here, we usually use plain title levels, I have no idea how it renders on the website.